packages/dd-trace/src/appsec/iast/iast-plugin.js

```diff
@@ -65,7 +65,7 @@ class IastPlugin extends Plugin {
       try {
         handler(message, name)
       } catch (e) {
-        iastLog.errorAndPublish(e)
+        log.error('[ASM] Error executing handler', e)
```

Comment:
To try to add more information in the message.

```suggestion
        log.error('[ASM] Error executing IAST plugin handler', e)
```